src/components/review/StorageReview.jsx

```diff
@@ -86,11 +87,15 @@ const DeviceRow = ({ disk }) => {
         const size = cockpit.format_bytes(devices[device].size.v);
         const request = requests.find(request => request["device-spec"] === device);
         const format = devices[device].formatData.type.v;
-        const action = (
+        let action = (
             request === undefined || request.reformat
                 ? (format ? cockpit.format(_("format as $0"), format) : null)
                 : ((format === "biosboot") ? format : _("mount"))
         );
+        // FIXME use better JS
```

Comment:
It's indeed weird to do the inline conditional and then do the if conditional to possibly overwrite it.
Since we have many conditions here, let's do them all in if clauses.

Reply:
Updated